atlasdb-config/src/test/java/com/palantir/atlasdb/http/AtlasDbHttpClientsTest.java

```diff
@@ -214,7 +227,7 @@ public void httpProxyIsConfigurableOnClientRequests() {
         clientWithHttpProxy.getTestNumber();
 
         proxyServer.verify(getRequestedFor(urlMatching(GET_ENDPOINT))
-                .withHeader(AtlasDbInterceptors.USER_AGENT_HEADER, WireMock.equalTo(DEFAULT_USER_AGENT)));
+                .withHeader(AtlasDbInterceptors.USER_AGENT_HEADER, WireMock.equalTo(DEFAULT_USER_AGENT_STRING)));
```

Comment:
nit, default should go to legacy

Reply:
(Clarified offline - this means naming; the behaviour is already in that direction.)